puppetdb/facts_test.go

```diff
@@ -9,7 +9,7 @@ import (
 // TestFactNames performs a test on the FactNames endpoint and verifies the expected response is returned.
 func TestFactNames(t *testing.T) {
 	// Test FactNames
-	setupGetResponder(t, "/pdb/query/v4/fact-names", "", "factnames-response.json")
+	setupGetResponder(t, factnames, "", "factnames-response.json")
```

Comment:
I had to check where this was defined.

This is also really lovely and I will copy it to orch:

```
const (
	factnames = "/pdb/query/v4/fact-names"
	facts     = "/pdb/query/v4/facts"
)
```